### PR TITLE
tiflow: adjust README files

### PR DESCRIPTION
**Pull request by niubell**
### What problem does this PR solve?

Issue Number: ref #3194

as the title says

### Check List 

Tests 

 - No code

Code changes

Side effects

Related changes

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Review by nongfushanquan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,80 +1,14 @@
...
-
-# Check sync progress
-$ mysql -h 127.0.0.1 -P 5000 -u root -e "SELECT COUNT(*) FROM test.sbtest1"
-```
-
-## Contributing
-
-Contributions are welcomed and greatly appreciated. See [CONTRIBUTING.md](./CONTRIBUTING.md)
-for details on submitting patches and the contribution workflow.
+## Introduction
```
Comment (lance6716) [inline README.md:9]:
```suggestion
## Introduction

```

File: README.md [hunk 2]
```diff
@@ -1,80 +1,14 @@
...
+**TiFlow** is a unified data replication platform around [TiDB](https://docs.pingcap.com/tidb/stable), including two main components: DM and CDC. DM supports full data migration and incremental data replication from MySQL/MariaDB into [TiDB](https://docs.pingcap.com/tidb/stable). CDC supports replicating change data to various downstreams, including MySQL protocol-compatible databases, message queues via the open CDC protocol and other systems such as local file storage. More details can be found in [DM README](./README_DM.md) and [CDC README](./README_CDC.md).
```
Comment (lance6716) [inline README.md:10]:
```suggestion
**TiFlow** is an unified data replication platform around [TiDB](https://docs.pingcap.com/tidb/stable), including two main components: DM and TiCDC. DM supports full data migration and incremental data replication from MySQL/MariaDB into [TiDB](https://docs.pingcap.com/tidb/stable). TiCDC supports replicating change data from TiDB to various downstreams, including MySQL protocol-compatible databases, message queues via the open CDC protocol and other systems such as local file storage. More details can be found in [DM README](./README_DM.md) and [TiCDC README](./README_CDC.md).
```

**Review by lance6716 — APPROVED** [review]

**Review by amyangfei — COMMENTED** [review]
The architecture path in readme_dm.md is not correct after migration
`architecture`

File: README.md (hunk 2 above)
Comment (niubell) [reply README.md:10]: `A unified` is correct because of having consonant sound.

Comment (niubell) [reply README.md:10]: TiCDC or CDC ? @nongfushanquan

File: README.md (hunk 1 above)
Comment (niubell) [reply README.md:9]: done

**Comment by niubell** [comment]
> 

done

**Comment by niubell** [comment]
> The architecture path in readme_dm.md is not correct after migration `architecture`

done

**Review by amyangfei — APPROVED** [review]

**Pull request merged**